velox/parse/ExpressionsParser.h

```diff
@@ -16,15 +16,25 @@
 #pragma once
 
 #include <set>
+#include "velox/core/PlanNode.h"
 #include "velox/core/QueryCtx.h"
 #include "velox/parse/IExpr.h"
 
-namespace facebook {
-namespace velox {
-namespace parse {
+namespace facebook::velox::parse {
 
-std::shared_ptr<const core::IExpr> parseExpr(const std::string& expr);
+/// Hold parsing options.
+struct ParseOptions {
+  // Retain legacy behavior by default.
+  bool preserveIdentifierCase = true;
```

Comment:
Do we need to expose all of these options? Can we limit this struct to just 'parseDecimalAsDouble' ?

Reply:
No need. Removed the extra options.